style.css

```diff
@@ -0,0 +1,232 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+body {
+  font-family: 'Poppins'
+}
+
```

Comment:
If you had 
*{
box-sizing: border-box;
}
This will fix the issue where the box with the magnifying glass doesn't align properly with the search bar,.
Here is some further documentation on it. https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing
It's a common thing to see in most css resets (css files dedicated to take out / change all the default css properties applied by the browser).

Other than that your npm clone looks really good : ) well done.